### ISSUE TITLE
support multiple item groups

### ISSUE THREAD

**Issue by aliasadidev** [aliasadidev/vscode-npm-gui#44]

**Comment by eburnette** [comment]
I ran into this one. I can't open my project because it has multiple item groups, some of which are conditional. The NuGet Package Manager GUI just spins and never finishes loading. I get an error about "More than one <ItemGroup> find."

**Comment by jobou363** [comment]
I have also the same problem because in have 3 items groups. It seems I can't regroup and use one. 

<ItemGroup>
    <PackageReference Include="System.Data.DataSetExtensions" Version="4.5.0" />
  </ItemGroup>
  <ItemGroup>
    <PackageReference Include="Test.Test" Version="1.3.2" />
  </ItemGroup>
  <ItemGroup>
    <Folder Include="My Project\" />
    <Folder Include="Extensions\" />
  </ItemGroup>

**Comment by mikes-gh** [comment]
This is unfortunate because this is the best tool out there.  But I am multi targeting net6 net7 as I suspect a few others are making this extension non-functional.  Mind you dependabot cant cope with this either (although it doesnt crash)

**Comment by davidsr2r** [comment]
Facing the same issue here, quite annoying!

**Comment by enny1170** [comment]
Same problem here in a MAUI Project. I can't change to singe ItemGroup because the others has conditions needed. Pls find a way to support multiple item groups

**Comment by Ghostbird** [comment]
Yeah, I tried this extension, but it's completely useless because of this odd restriction. I did a brief scan of the code, and I think it's because the package reference elements are identified by index. That's a strange design choice that introduces all kinds of seemingly arbitrary restrictions. My first take on writing something like this, is that the package reference elements should be identified by their `Include` value.

For a quick estimate, I checked the repositories I currently have cloned on my machine, and out of 20 projects, only 2 had less than 2 `ItemGroup`s. But those two were projects that only defined a few models and were referenced by other projects in the same solution. So for those repositories the extension still didn't work, as the referencing project file did have multiple `ItemGroup`s.

Conclusion: Because of this arbitrary restriction, this extension cannot work in any of my normal use cases.

**Comment by aliasadidev** [comment]
I started it

**Comment by christophe-calmejane** [comment]
Same issue here, I need to use conditionals on many projects, they fail to load in the extension.
